x-pack/plugins/cases/server/common/utils.test.ts

```diff
@@ -470,14 +466,13 @@ describe('common utils', () => {
       `);
     });
 
-    it('inserts missing connector', () => {
+    it('does not transform the connector to the ES schema, it should leave it with the id field', () => {
```

Comment:
`flattenCaseSavedObject` no longer does transformations. The service layer does it.

Reply:
nit: I think the title is a bit misleading as the purpose of this test is not to check the ES schema.

Reply:
Any suggestion on what to use for the title? Do we need the test anymore or should I remove it?